PayPalCheckout/src/main/java/com/paypal/android/checkout/pojo/ApprovalPojos.kt

```diff
@@ -0,0 +1,31 @@
+package com.paypal.android.checkout.pojo
+
+import com.paypal.checkout.approve.Approval
+
+
+data class Approval(
+    val approvalData: ApprovalData? = null,
+) {
+    internal constructor(approval: Approval) : this(
+        approvalData = ApprovalData(approval.data),
+    )
+}
+
+
+data class ApprovalData(
```

Comment:
Can we doc string the elements on ApprovalData? I understand that a merchant would use the orderId here to capture/authorize with the orders API, but I don't quite understand why we are surfacing all of this other data (like buyer and cart), and what a merchant should be using it for at this point in the flow.

Reply:
@sarahkoop this object is was native checkout returns wether if you do a server side integration, or by passing an `order`. Some merchants have asked for this info on native checkout, so that they dont have to do more api calls to get the information.